### ISSUE TITLE
IDISP003 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#52]
```cs
namespace RoslynSandbox
{
    using System.IO;

    public class Foo
    {
        public Foo()
        {
            if (TryGetStream(out var stream))
            {
            }
        }

        private static bool TryGetStream(out Stream stream)
        {
            stream = File.OpenRead(string.Empty);
            return true;
        }
    }
}
```
